### DISCUSSION TITLE
cannot download videos from Twitter and Facebook

### DISCUSSION THREAD

**Discussion by kirkwoodlao** [ErrorFlynn/ytdlp-interface#100] [Q&A]
After reinstalling the system (Win10 22H2), I cannot download videos from Twitter and Facebook, how to solve it? thanks

**Comment by ErrorFlynn** [comment]
Go to `Settings -> Updater`, select the `Nightly` release channel, and click `Update yt-dlp`. I have verified that this fixes Twitter downloads, and I imagine it fixes Facebook too.
